markdown/3.x/en/api/ff-filter-cloud.api.md

```diff
@@ -0,0 +1,13 @@
+## `ff-filter-cloud`
+___
+### Properties
```

Comment:
Specifications seems to have been [updated](https://github.com/FACT-Finder-Web-Components/feature-lab/blob/ff-filter-cloud/specs/ff-asn-selected-filter-cloud/ff-asn-selected-filter-cloud.md#api) since.

Also, an example of what _a comma separated list_ can look like should be helpful to users.